contentScript.js

```diff
@@ -62,6 +60,7 @@ window.addEventListener("SetupComplete", function() {
   });
   window.dispatchEvent(new CustomEvent("InjectedScriptEval", {detail: `
     lowLag.load("${chrome.runtime.getURL("/assets/retry.wav")}", "retry");
+    game.pulseHaxLogo = "${chrome.runtime.getURL("/assets/icon.ico")}"
```

Comment:
This is an example of a code review. Doing pull requests makes these reviews way easier. If I weren't lazy I would also ask you to review my prs through github but I've been asking you through discord instead.

Reply:
If I had to guess you forgot to commit your changes to home to enable/disable the setting. I'm fine to merge this but I don't think it does everything you want it to.

Reply:
I just wanted to get it to work on ingameport, Im not all that much familiar with the html stuff and also something I noticed is that the settings wont sync between the game port and the actual popups because they use different variables and I have zero clue on how to sync them up